Assets/Scripts/Internal/DaggerfallInterior.cs

```diff
@@ -940,7 +940,7 @@ private void AddPeople(PlayerGPS.DiscoveredBuilding buildingData)
                     }
 
                     // Set position
-                    DaggerfallBillboard dfBillboard = go.GetComponent<DaggerfallBillboard>();
+                    Billboard dfBillboard = go.GetComponent<Billboard>();
```

Comment:
I think this part of the code should probably be moved out of the "if".
 
For example, Vanilla Enhanced's animated people uses custom flat gameobjects (see line 929) to insert the custom billboard. After this PR, I'll change its prefabs to use the new Billboard interface, and it'd be best if `SetRMBPeopleData` was called on custom billboards too

Reply:
Looking at it, not sure about position. I've moved the set people data call. Will let Interkarma comment on the position adjustment because I am not sure either.

Reply:
I note that `ImportCustomFlatGameobject` calls its own `AlignToBase` handler which does a position adjustment. If we also did that outside of the `if` block here then position would be adjusted twice for imported replacements.

I think it's fine the way it is.